db/schema.sql

```diff
@@ -141,18 +141,18 @@ CREATE TABLE report_aggregations(
 CREATE INDEX report_aggregations_aggregation_job_id_index ON report_aggregations(aggregation_job_id);
 CREATE INDEX report_aggregations_client_report_id_index ON report_aggregations(client_report_id);
 
--- Information on aggregation for a single batch unit. This information may be incremental if the
--- VDAF supports incremental aggregation.
-CREATE TABLE batch_unit_aggregations(
+-- Information on aggregation for a single batch. This information may be incremental if the VDAF
+-- supports incremental aggregation.
+CREATE TABLE batch_aggregations(
     id                    BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,  -- artificial ID, internal-only
     task_id               BIGINT NOT NULL,     -- the task ID
-    unit_interval_start   TIMESTAMP NOT NULL,  -- the start of the batch unit
+    batch_identifier      BYTEA NOT NULL,      -- encoded query-type-specific batch identifier (corresponds to identifier in BatchSelector)
```

Comment:
I lean away from this for a couple of reasons:
1) This is an ID. I think JSON is more useful for structured data.
2) JSON is not deterministically encoded. Using JSON would open us up to potential bugs around differences in encoding, since this column is used for lookups/indexing.
3) The only way we'll need to expand the encoding is when we switch to a new version of the DAP spec; but switching DAP spec versions is a breaking change, so we can make a breaking change here. (that is, IMO we aren't at risk of "painting ourselves into a corner" by using this encoding rather than a more flexible one)

TBH, I'm fine leaving the interpretation up to the query type -- there are other places that implicitly do so. If we want to have some kind of indicator, we could prefix the batch identifier with a query type byte. Thoughts?

(slightly off-topic: I also considered encoding time interval's batch identifier as the start time of the interval; this doesn't lose data since the duration of the interval will always match `time_precision`. but this would have required making the `time_precision` available to anywhere that is reading batch identifiers, since the logical batch identifier is indeed an interval.)

In the longer run, I'd like it if "batch_identifier" could just be a `batch_id` in all cases. It's sort of awkward to use `batch_identifier` and `batch_id` as separate concepts, since "id" is short for "identifier". (maybe there's a better name possible?) But we'd need some way to deal with the time interval's batch interval, and I can't think of a way to do so that doesn't introduce more complexity than we save.